src/pages/display/index.js

```diff
@@ -29,23 +31,38 @@ export default class Display extends Component {
 
   componentWillMount = async () => {
     this.context.setPath(window.location.pathname)
-    const currentWallet = window.location.pathname.split('/')[2]
-    console.log('current wallet', currentWallet)
+
+    fetch(`https://api.tzkt.io/v1/accounts/${this.state.wallet}`)
```

Comment:
can you move this API call to `src/data/api.js` ? or maybe tzkt.js is more appropriate. I'm trying to make the pages and the components as "dumb" as possible so we can easily unit test them. the logic of data handling and any type of filtering that needs to happen (or not) I would rather keep outside the page itself. Is that okay?

Reply:
Sounds good to me, I'll do that as soon as I get home.